src/dotnet/APIView/APIViewWeb/Repositories/DevopsArtifactRepository.cs

```diff
@@ -71,5 +87,16 @@ private string GetArtifactRestAPIForRepo(string repoName)
             }
             return downloadArtifactRestApi;
         }
+
+        public async Task RunPipeline(string pipelineName, string reviewDetails, string originalStorageUrl)
+        {
+            SetDevopsClientHeaders();
+            //Create dictionary of all required parametes to run tools - generate-<language>-apireview pipeline in azure devops
+            var reviewDetailsDict = new Dictionary<string, string> { { "Reviews", reviewDetails }, { "APIViewUrl", _hostUrl }, { "StorageContainerUrl", originalStorageUrl } };
+            var pipelineParams = new Dictionary<string, Dictionary<string, string>> { { "templateParameters", reviewDetailsDict } };
+            var stringContent = new StringContent(JsonConvert.SerializeObject(pipelineParams), Encoding.UTF8, "application/json");
```

Comment:
Why do we use NewtonSoft here, but System.Text.Json in ReviewManager? Ideally we could use System.Text.Json everywhere.

Reply:
Updated to System.Test.Json